tools/train.py

```diff
@@ -103,6 +105,12 @@ def main():
     if args.resume:
         cfg.resume = args.resume
 
+    # resume is determined in this priority: resume from > auto_resume
+    cfg.resume = args.resume
```

Comment:
This line will cause resume conflict. Please refer to the logic in https://github.com/open-mmlab/mmdetection/pull/9287

Reply:
Done. I have removed this line.